Trappist/src/Promact.Trappist.Test/Tests/TestsRepositoryTest.cs

```diff
@@ -131,16 +132,40 @@ public TestsRepositoryTest(Bootstrap bootstrap) : base(bootstrap)
         [Fact]
         public async Task UpdateTestName()
         {
-            var test = CreateTest("AOT 669");
+            var test = CreateTest("Computer");
             string userName = "suparna@promactinfo.com";
             ApplicationUser user = new ApplicationUser() { Email = userName, UserName = userName };
             await _userManager.CreateAsync(user);
             var applicationUser = await _userManager.FindByEmailAsync(user.Email);
             await _testRepository.CreateTestAsync(test, applicationUser.Id);
-            test.TestName = "MCKV";
+            test.TestName = "Computer";
```

Comment:
magic string used at more than one place should be included in a string constant.